src/picongpu/include/simulation_defines/param/ionizationEnergies.param

```diff
@@ -49,10 +51,26 @@ namespace SI
      * );
      *
      * Do not forget to change the weight of the ion and the atomic numbers in
-     * their respective `.param` file */
+     * their respective `.param` file
+     */
     PMACC_CONST_VECTOR(float_X,1,IONIZATION_EFIELD,
         5.14e11
     );
 
+    /* atomic unit for energy:
+     * 1 Rydberg = 27.21 eV --> converted to Joule
+     */
+    const float_X ATOMIC_UNIT_ENERGY = 4.36e-18;
```

Comment:
the values here must be `float_64` if in SI and they should be normalized in a `ionizationEnergies.unitless` file before being used in the code.